lib/menu.rb

```diff
@@ -0,0 +1,26 @@
+class Menu
+  
+  attr_reader :menu
+  
+  def initialize
+    @menu = [
+      { :item => 1, :dish => "Mattar Paneer", :price => 12.50 },
+      { :item => 2, :dish => "Black Daal", :price => 7.50 },
+      { :item => 3, :dish => "Raita", :price => 3.50 },
+      { :item => 4, :dish => "Garlic Naan", :price => 3.50 }
+    ]
+  end
```

Comment:
Hashes are most useful where you have a key value pair - eg. if the only information you needed to record was the item name and price, a simple hash could look like:

`@menu = { "1. Mattar Paneer" => 12.50, "Black Daal" => 7.50 } `

@menu[item_name] will return the cost of that item. 

For data requiring more than two values, where key/value isn't enough, I would suggest defining a new object, for example:

```
class Dish
    def initialize(name, price)
        @name = name
        @price = price
    end

    # getter methods for retrieving information
end
```

This way, you could work with an array of Dishes inside your Menu class, eg. @menu = [Dish.new(args), Dish.new(args), etc.]

Singular dish data can be accessed with @menu[0].name, etc. Which can be quite a bit easier than traversing a complex hash! 


end